src/FileConventions.Test/DummyFiles/DummyCIWithoutLatestTag.yml

```diff
@@ -0,0 +1,120 @@
+          npx commitlint --version
+      - name: Install yarn
+        run: |
+          npm install -g yarn
+          yarn add --dev jest typescript ts-jest @types/jest
+      - name: Install commitlint
+        run: |
+          npm install conventional-changelog-conventionalcommits
+          npm install commitlint@latest
+      - name: Print versions
+        run: |
+          git --version
+          node --version
+          npm --version
+          npx commitlint --version
+      - name: Run tests to validate our plugins
+        run: yarn jest
+
+  sanitycheck:
+    name: Sanity check
+    runs-on: ubuntu-22.04
+    container:
+      image: "ubuntu:22.04"
+    steps:
+      - name: Install required dependencies
+        run: |
+          apt update && apt install -y sudo
+          sudo apt install -y git
+          sudo DEBIAN_FRONTEND=noninteractive apt install --yes npm
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          # needed because of commit-lint, see https://github.com/conventional-changelog/commitlint/issues/3376
+          fetch-depth: 0
+      # workaround for https://github.com/actions/runner/issues/2033
+      - name: ownership workaround
+        run: git config --global --add safe.directory '*'
+      - name: Print versions
+        run: |
+          git --version
+          node --version
+          npm --version
+      - name: Validate current commit (last commit) with commitlint
+        if: github.event_name == 'push'
+        run: ./commitlint.sh --from HEAD~1 --to HEAD --verbose
+      - name: Validate PR commits with commitlint
+        if: github.event_name == 'pull_request'
+        run: ./commitlint.sh --from ${{ github.event.pull_request.head.sha }}~${{ github.event.pull_request.commits }} --to ${{ github.event.pull_request.head.sha }} --verbose
+      - name: Install dotnet sdk
+        run: sudo apt install -y dotnet6
+      - name: Check all files end with EOL
+        run: dotnet fsi scripts/eofConvention.fsx
+      - name: Check all .fsx scripts have shebang
+        run: dotnet fsi scripts/shebangConvention.fsx
+      - name: Check there are no mixed line-endings in any files
+        run: dotnet fsi scripts/mixedLineEndings.fsx
+      - name: Install prettier
+        run: npm install prettier@2.8.3
+      - name: Change file permissions
+        # We need this step so we can change the files using `npx prettier --write` in the next step.
+        # Otherwise we get permission denied error in the CI.
+        run: sudo chmod 777 -R .
+      - name: Run "prettier" to check the style of our TypeScript code
+        run: |
+          sudo npx prettier --quote-props=consistent --write './**/*.ts'
+          # Since we changed file modes in the previous step we need the following command to
+          # make git ignore mode changes in files and doesn't include them in the git diff command.
+          git config core.fileMode false
+          git diff --exit-code
```

Comment:
@realmarv woah there's no need to have these test files so big, use the simplest YML that you can think of, that just prints hello world or something

Reply:
Done